<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-size: 10pt;
}

QDialog {
	
	background-color: rgb(0, 0, 0);
}

QLabel {
	color: rgb(255, 246, 0);
}

QLineEdit, QComboBox, QDateEdit {
	color: rgb(255, 246, 0);
	
	background-color: rgb(0, 0, 0);
	border: 1px solid #fff600;
}

QPushButton {
	background-color: rgb(0, 0, 0);
	color: rgb(255, 246, 0);
	border: 3px solid #fff600;
}

QPushButton:hover {
	background-color: #fff600;
	color: black;
}

#frame {
	
	background-color: rgb(255, 255, 255, 80);
	border-radius: 4px;
}

QRadioButton {
	color: #fff600;
}

QRadioButton::indicator::unchecked {
	
	
	background-color: rgb(0, 0, 0);
	border: 1px solid #fff600;
	border-radius: 2px;
}

QRadioButton::indicator::checked {
	
	
	background-color: #fff600;
	border: 1px solid black;
	border-radius: 2px;
}

</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>371</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="lblAirline">
        <property name="text">
         <string>Airline:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboAirline"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Name: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtFname"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Family: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtLname"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>National Code: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtNationalCode"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Personnel Code: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPersonnelCode"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Birth Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dtBirthDate"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Employment Date: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dtEmploymentDate"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Degree: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QRadioButton" name="radio3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
